codecov_cli/commands/report.py

```diff
@@ -13,15 +13,6 @@
 @click.option(
     "--code", help="The code of the report. If unsure, leave default", default="default"
 )
-@click.option(
```

Comment:
this is a breaking change. I think that users that run the CLI in a `workflow_dispatch` need this

Reply:
I just don't know if it makes sense for the create-report command

Reply:
ahhh I see, that does make sense. however, we should probably for now make it a `deprecated` field